tests/integration/release/test_release.py

```diff
@@ -103,6 +105,19 @@ def test_full_release_noising(
     ]
 
     if engine == DASK_ENGINE:
+        cluster = LocalCluster(
+            n_workers=int(os.environ["SLURM_CPUS_ON_NODE"]),
```

Comment:
This does require we run release tests on slurm. I'm sure we'd never not do that anyway but something to note